src/extension.ts

```diff
@@ -36,6 +36,46 @@ export function activate(context: vscode.ExtensionContext) {
       const missingKeysInDotEnv = configTsKeys.filter(
         x => !parsedDotEnvKeys.includes(x)
       )
+      const hiddenPattern = /hidden\([\n\r\s]*('|").*("|')[\n\r\s]*\)/g
+      const hiddenTsKeys =
+        text
+          .match(hiddenPattern)
+          ?.map(x => x.slice(7, -1).trim().slice(1, -1)) ?? []
+
+      const diagnostics: vscode.Diagnostic[] = []
+
+      // ============== Underline keys with unsafe default values in config.ts file ==============
+      for (const hiddenKey of hiddenTsKeys) {
+        const safeDefaultValue =
```

Comment:
What if the hidden value is of different type than `string`?

`bool` - We don't want to check I guess
`json` - `{}`?
`number` - `0`? 😅 
`custom` - Not our business I guess

Reply:
Discussed in person - only strings will be checked